apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -102,8 +102,11 @@ const nameFormatter = (projectName, {rowData}) => {
 };
 
 const actionsFormatter = (actions, {rowData}) => {
+  const unpublishableTypes = ['calc', 'eval', 'weblab'];
```

Comment:
We already have constants to tell us which projects are publishable. Check out sharedConstants.js which is generated by https://github.com/code-dot-org/code-dot-org/blob/ffc01457a8474e3221c11f8fd046794b8cb0df5a/lib/cdo/shared_constants.rb#L71-L99 . I would be better to reuse the existing list. 

Also, does this new logic account for the fact that some projects can only be published for over 13 or with the appropriate sharing setting?